test/test_rw_fortran.F90

```diff
@@ -275,8 +275,7 @@ END FUNCTION real_eq
          
          IF (absdiff > max_absdiff) max_absdiff = absdiff
          IF (reldiff > max_reldiff) max_reldiff = reldiff
-
-         IF( .NOT.real_eq(obuf(j), cbuf(j), 125) ) THEN
+         IF( .NOT.real_eq(obuf(j), cbuf(j), 2000) ) THEN
```

Comment:
I was getting random failures of tests when comparing the difference between compress/uncompressed data. This might have been undetected before because we were not returning an exit code when the program fails, and we were not checking for an exit code in the Makefile.